tests/baselines/reference/importTypeInJSDoc.types

```diff
@@ -37,14 +37,14 @@ export = MyClass;
  */
 
 let a = /** @type {Foo} */(/** @type {*} */(undefined)); 

```

Comment:
This seems to contradict the intention of this test, which I think was to translate `let a = undefined as any as Foo` to JS. If there's a new way to translate this, I think it's fine to change the source, but I don't think the type of `a` should change.